modules/fscloud/variables.tf

```diff
@@ -132,7 +132,7 @@ variable "addons" {
     vpc-block-csi-driver      = optional(string)
```

Comment:
you will want to add the new addon to this object as well, to match root module input